src/utils/tree_node.C

```diff
@@ -457,25 +460,30 @@ unsigned int TreeNode<N>::n_active_bins() const
 
 
 template <unsigned int N>
-const Elem* TreeNode<N>::find_element(const Point& p, const std::set<subdomain_id_type> *allowed_subdomains) const
+const Elem*
+TreeNode<N>::find_element
+(const Point& p,
+ const std::set<subdomain_id_type> *allowed_subdomains,
+ Real relative_tol) const
 {
   if (this->active())
     {
       // Only check our children if the point is in our bounding box
       // or if the node contains infinite elements
-      if (this->bounds_point(p) || this->contains_ifems)
+      if (this->bounds_point(p, relative_tol) || this->contains_ifems)
```

Comment:
As long as we're talking about removing that piece of `find_element_in_children`... I would vote for removing find_element_in_children!  I think it makes things less clear.  I would prefer to just have one recursive function that does what it's supposed to do...